Makefile

```diff
@@ -98,17 +97,17 @@ install.tools: .install.golint .install.govet .install.gitvalidation
 # golint does not even build for <go1.5
 .install.golint:
 ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
-	go get github.com/golang/lint/golint
+	go get -u github.com/golang/lint/golint
 endif
 
 # go vet is now included in >=go1.5, so no need to get it.
 .install.govet:
-ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
-	go get golang.org/x/tools/cmd/vet
+ifneq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
```

Comment:
I'm not seeing where go1.4 is supported. What are you saying? that go1.4 ought to be allowed, but isn't? or that govet now support go1.4?